frontend/src/components/pages/RegistrantExperience/RegistrationResult/index.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useEffect } from "react";
+import { Text, Flex, VStack, HStack } from "@chakra-ui/react";
+import { CartItem, EdlpChoice } from "../../../../types/RegistrationTypes";
+
+import {
+  calculateTotalPrice,
+  mapCampToCartItems,
+} from "../../../../utils/RegistrationUtils";
+import RegistrationInfoCard from "./RegistrationInfoCard";
+import {
+  cardBoldStyles,
+  regularTextStyles,
+  resultTextStyles,
+  resultTitleStyles,
+  subheadingStyles,
+} from "./textStyles";
+import { CampResponse, CampSession } from "../../../../types/CampsTypes";
+import { RegistrantExperienceCamper } from "../../../../types/CamperTypes";
+import CamperAPIClient from "../../../../APIClients/CamperAPIClient";
+
+const NoSessionDataFound = (): React.ReactElement => {
+  return (
+    <>
+      <Text textStyle={resultTitleStyles}>No checkout session data found</Text>
+      <Text
+        mt={{ sm: "6px", md: "32px", lg: "20px" }}
+        textStyle={resultTextStyles}
+      >
+        Cannot load session data. If you expect to see a success message, please
+        check your email. You should receive a payment receipt within 15
+        minutes, including more details about your registration.
+      </Text>
+    </>
+  );
+};
+
+const PaymentSummaryList = ({
+  items,
+}: {
+  items: CartItem[];
+}): React.ReactElement => {
+  return (
+    <VStack w={{ base: "100%", lg: "35vw" }} spacing={10}>
+      <VStack w="100%" overflowY="auto" h={{ lg: "35vh" }} spacing={8}>
+        {/* TODO switch to sum of EDLP, non-EDLP items */}
```

Comment:
not sure what this is referring to exactly but did you want to address it in this PR?

Reply:
nah, I handle it in the EDLP fixes PR